crops/command_line/crops-renumber.py

```diff
@@ -73,9 +76,14 @@ def main():
                 try:
                     newstructure = cop.renumber_pdb(seqset[seqname], structure)
                 except (AttributeError, IndexError) as e:
-                    logger.warning('Something has gone wrong during renumbering:\n{}'.format(e))
-                    logger.info('Attempting Needleman-Wunsch...')
-                    newstructure = cop.renumber_pdb_needleman(seqset[seqname], structure)
+                    if args.force_alignment is True:
+                        logger.warning('Something has gone wrong during renumbering:\n{}'.format(e))
+                        logger.info('Attempting Needleman-Wunsch...')
+                        newstructure = cop.renumber_pdb_needleman(seqset[seqname], structure)
+                    else:
+                        logger.critical('Something has gone wrong during renumbering:\n{}'.format(e))
+                        raise RuntimeError from e
```

Comment:
What's the point of catching an exception to raise another one? I think in this case it's fine just to log the critical message as you've done above and exit gracefully without raising an exception.